.github/ISSUE_TEMPLATE/dev-ticket.md

```diff
@@ -0,0 +1,33 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Story
+<!--- Describe the benefits of this change, ideally in the following format: -->
+<!--- As a < type of user >, I want < some goal > so that < some reason >. -->
+
+## Rationale
+<!--- Provide the relevant background information to udnerstand why this change is needed -->
+<!--- This is where you can provide more details thathelp the reader understand why you want to make this change -->
+
+## Implementation details
+<!--- Explain the steps you will take to implement this change -->
```

Comment:
How about - Outline the implementation route you plan to take 
(Because at this point, the exact steps 'you will take' may not be 100% clear. But if they can't write anything here, then we know we'll need to spike).

Reply:
```suggestion
<!--- Explain the steps you will take to implement this change. If you do not have specific steps yet, write your implementation ideas or outline general approach -->
```